views/regional_statistics_view/20190913123830.sql

```diff
@@ -0,0 +1,43 @@
+SELECT region_id, land_area, eez_area, ts_area,
+  (COALESCE(pa_land_area,0) + COALESCE(pa_marine_area,0)) AS pa_area,
+  pa_land_area, pa_marine_area, pa_eez_area, pa_ts_area,
+  ((COALESCE(pa_land_area,0) + COALESCE(pa_marine_area,0)) /
+    (land_area + COALESCE(eez_area, 0) + COALESCE(ts_area,0))*100) AS percentage_pa_cover,
+  (COALESCE(pa_land_area,0) / land_area * 100) AS percentage_pa_land_cover,
+  (COALESCE(pa_marine_area,0) / marine_area * 100) AS percentage_pa_marine_cover,
+  (COALESCE(well_connected_area,0) / (land_area + marine_area) * 100) AS percentage_well_connected,
+  (COALESCE(importance_area,0) / (land_area + marine_area) * 100) AS percentage_importance,
+  (COALESCE(pame_pa_land_area,0) / COALESCE(pa_land_area,0) * 100) AS pame_percentage_pa_land_cover,
```

Comment:
I will check for the divisions by zero, not sure if SQL complains in the same way as Rails does...